src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -56,13 +56,16 @@ def register_options(cls, register):
             "--timeout-default",
             type=int,
             advanced=True,
-            help="The default timeout (in seconds) for a test target if the timeout field is not set on the target.",
+            help=(
+                "The default timeout (in seconds) for a test target if the `timeout` field is not "
+                "set on the target."
+            ),
         )
         register(
             "--timeout-maximum",
             type=int,
             advanced=True,
-            help="The maximum timeout (in seconds) that can be set on a test target.",
+            help="The maximum timeout (in seconds) that may be used on a `python_tests` target.",
```

Comment:
I'm not sure if I should go into more detail. I'm concerned this suggests that we'll error. Really, we'll just call `max(tgt, global_option)`.